modules/drivers/oracle/test/metabase/test/data/oracle.clj

```diff
@@ -77,6 +89,38 @@
           session-schema
           (tx/db-qualified-table-name database-name table-name)))
 
+(defonce ^:private oracle-test-dbs-created-by-this-instance (atom #{}))
+
+(defn- destroy-test-database-if-created-in-different-session
+  "For Oracle, we have a randomly selected `session-schema`, in order to allow different test runs to proceed
+  independently. If we happen to have a test-database already in our app DB, that was created under a different session,
+  then we should just destroy it, since we won't be able to resync it anyway."
```

Comment:
is it not better to have the creator clean up after themselves? I'm not following 100% how this issue arises.

Reply:
This line was written quite a while ago, now, but I had a fairly long Slack discussion with @camsaul  about it.  I'll find the context there and update the docstring accordingly.